terrat_runner/workflow_step_terrateam_ssh_key_setup.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import os
+import workflow
+import subprocess
+
+def run(state, config):
+    logging.info('TERRATEAM_SSH_KEY : SETUP')
+    env = state.env
+
+    # Create the ssh directory
+    ssh_dir_path = "/root/.ssh/"
+    if not os.path.exists(ssh_dir_path):
+        os.makedirs(ssh_dir_path)
+
+    # Iterate through environment variables that start with TERRATEAM_SSH_KEY
+    ssh_keys = [(key, val) for key, val in env.items() if key.startswith('TERRATEAM_SSH_KEY')]
```

Comment:
I think this loop should be pulled out since it is used in two places.  And the constant `TERRATEAM_SSH_KEY` can be pulled out to its own var